Time off tracker WEB/Controllers/HomeController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Time_off_tracker_WEB.Models;
+
+namespace Time_off_tracker_WEB.Controllers
+{
+    [ApiController]
+    [Route("/[controller]")]
+    public class HomeController : ControllerBase
```

Comment:
We don't need this, for now, remove it

Reply:
It's better to add Exception handler via attribute - https://metanit.com/sharp/aspnet5/17.2.php
Then Create a BaseContoller with this attribute

Reply:
Create ExceptionFilterAtribute without realisation. I will add the error output to the log in the logging implementation branch.